package.json

```diff
@@ -794,7 +794,7 @@
         "@azure/ms-rest-js": "^2.2.1",
         "@microsoft/vscode-azext-azureappservice": "^0.7.4",
         "@microsoft/vscode-azext-azureutils": "^0.3.4",
```

Comment:
We should update the other two packages from `vscode-azuretools` at the same time. Otherwise, you'll have a mix of 0.3.7 and 0.4.0 of `@microsoft/vscode-azext-utils`.

Reply:
I will hold off on updating app service since I don't want to incorporate the kudu changes yet.

Reply:
Ok. Is that something we want to do before release?

Reply:
Probably not for the v1.5 release. Unless it's super simple. I'll talk to Nathan about it.